hpx/runtime/parcelset/encode_parcels.hpp

```diff
@@ -202,6 +190,7 @@ namespace hpx
 #endif
 
                             LPT_(debug) << ps[i];
+                            archive.set_splitted_gids(ps[i].splitted_gids());
                             archive << ps[i];
```

Comment:
'splitted' is not a word in English. The verb 'split' is conjugated as 'to split, have split, had split'.